test/utils.py

```diff
@@ -815,3 +815,24 @@ def log_contains(self, re_string, times=None):
             else:
                 match_count = len(re.findall(re_string, content))
                 assert match_count == times
+
+    """
+    Run pgbouncer instance with provided config and restore previous config in after execution
```

Comment:
Should this description be part of the function?

Reply:
Definitely! I'll move it inside the function

Reply:
Fixed